src/Resources/Resources/ActiveDirectory/Cmdlets/NewAzureADServicePrincipalCommand.cs

```diff
@@ -314,17 +314,14 @@ private void CreateSimpleServicePrincipal()
                     var application = ActiveDirectoryClient.CreateApplication(appParameters);
                     ApplicationId = application.ApplicationId;
                     WriteVerbose(string.Format("No application id provided - created new AD application with application id '{0}'", ApplicationId));
+                    printPassword = true;
                 }
             }
```

Comment:
In the case where we don't create credentials, we should print an additional verbose message saying "No new secret was created. This ServicePrincipal will use the password associated with application '{applicationId}'"

Reply:
Fixed